platform/broadcom/sonic-platform-modules-ruijie/common/script/device_i2c.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+import sys
+import click
+import os
+import commands
+import time
+import mmap
+from  ruijieconfig import *
```

Comment:
lgtm [py/non-iterable-in-for-loop
Not interesting about this!

Reply:
Rather than `from  ruijieconfig import *`, you can instead `import ruijieconfig` and reference everything as `ruijieconfig.<name>`. This will fix the LGTM alert.